### DISCUSSION TITLE
Scratch Addons icon/logo discussion

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#305] [Discussion]
🚀 **Rocket**
![scratchaddons 360x360](https://user-images.githubusercontent.com/17484114/93685911-27000e00-fa89-11ea-82ef-baa51f2f67f9.png)

👀 **Eyes**
![2020_09_19_085304](https://user-images.githubusercontent.com/17484114/93685901-0c2d9980-fa89-11ea-914f-24efb34d00fa.png)

**Screenshots**
Tampermonkey vs Scratch Messaging Extension vs both logos
![image](https://user-images.githubusercontent.com/17484114/93685933-4e56db00-fa89-11ea-8b4e-9781f8c73357.png)
![image](https://user-images.githubusercontent.com/17484114/93685960-78100200-fa89-11ea-9adf-897ba41a50c2.png)
![image](https://user-images.githubusercontent.com/17484114/93685974-8827e180-fa89-11ea-8a31-bee37fd5093f.png)

**Comment by WorldLanguages** [comment]
Tagging some active contributors please tell me what you think:
@BoomerScratch @apple502j @Hans5958 @GDUcrash @JeffaloBob @GrahamSH-LLK @NoobTracker @Explosion-Scratch @mxmou

**Comment by azureitte** [comment]
the pixelated borders and the color difference of the new icon are results of my stupidity, it won't be present in the actual icon.

**Comment by Explosion-Scratch** [comment]
The third one probably.

**Comment by WorldLanguages** [comment]
@Explosion-Scratch "The third one" can mean too many things, please add a reaction to avoid confusion :P

**Comment by NoobTracker** [comment]
I prefer the first one, rocket.

**Comment by jeffalo** [comment]
ok ok but what if, right we made the logo look like a cat and a Lego piece

**Comment by GrahamSH-LLK** [comment]
I 100% prefer the rocket. The circle makes the logo feel "complete" to me.

**Comment by Hans5958** [comment]
Keep in mind that for the lego-shaped logo, I need to give an outline for the header. I prefer the ones that are gradient-coloured by GDUcrash for that.

There is still a need to have an SVG version of the logo, instead of depending with the PNG one.

**Comment by azureitte** [comment]
In the old logo, you can't even see the puzzle piece, it's so small, it looks like a rectangle. Plus, the asymmetry...

**Comment by mxmou** [comment]
> In the old logo, you can't even see the puzzle piece, it's so small, it looks like a rectangle. Plus, the asymmetry...

The asymmetry isn't very noticeable, but I don't like the incomplete S.

**Comment by jeffalo** [comment]
> In the old logo, you can't even see the puzzle piece, it's so small, it looks like a rectangle. Plus, the asymmetry...

![image](https://user-images.githubusercontent.com/40470736/93708169-646da580-fb34-11ea-9a74-1150af06e781.png)

**Comment by NoobTracker** [comment]
@JeffaloBob arrrgh everyone will die

**Comment by jeffalo** [comment]
> @JeffaloBob arrrgh everyone will die

![image](https://user-images.githubusercontent.com/40470736/93712178-824b0280-fb54-11ea-830f-05de40c10b3d.png)

**Comment by NoobTracker** [comment]
@JeffaloBob arrrgh _**everyone**_ will die

**Comment by jeffalo** [comment]
> @JeffaloBob arrrgh _**everyone**_ will die

![image](https://user-images.githubusercontent.com/40470736/93713448-3ef49200-fb5c-11ea-8f1d-28fbae7eb319.png)

**Comment by NoobTracker** [comment]
arrrch **_everyone_** will die because markdown doesn't have underline

**Comment by GrahamSH-LLK** [comment]
_**<ins>Hello!</ins>**_

**Comment by NoobTracker** [comment]
How?

**Comment by mxmou** [comment]
@NoobTracker `<ins></ins>`

**Comment by NoobTracker** [comment]
<ins>Ah thanks</ins>

**Comment by Explosion-Scratch** [comment]
Wat, you can use html in github markdown?!
<details>
    <summary>Test</summary>
    <p>Lorem Ipsum testing htmlsum </p>
</details>

**Comment by Explosion-Scratch** [comment]
Whoa :O What about inline styling:
<h1 style='background: red; border-radius: 5px; color: white; text-shadow: 2 2 0 0 black;'>Testing</h1>

**Comment by Explosion-Scratch** [comment]
I guess not :(

**Comment by jeffalo** [comment]
wait whattt <a href="https://games.jeffalo.net">asdf</a>

sorry for going off topic but why is this a thing in GitHub

**Comment by WorldLanguages** [comment]
Lol this went too offtopic. Please use the Discord server for that :P

**Comment by ZenithRogue** [comment]
![scratchaddon-01](https://user-images.githubusercontent.com/11393734/95667876-a2595a80-0b31-11eb-8f18-b30ad6abc21c.png)
